Tests/AppTests/Application+Test.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Application+Test.swift
+//  AppTests
+//
+//  Created by  LaptopGCampos on 7/13/18.
+//
+
+@testable import App
+import Vapor
+
+extension Application {
+    
+    struct EmptyContent: Content {}
+    
+    static func testableWithRevertedMigrations() throws -> Application {
+        return try testable(arguments:  ["vapor", "revert", "--all", "-y"])
+    }
+    
+    static func testable(arguments: [String] = []) throws -> Application {
+        var config = Config.default()
+        var services = Services.default()
+        var enviroment = Environment.testing
+        enviroment.arguments = arguments
+        try testConfigure(&config, &enviroment, &services)
+        let app = try Application(config: config, environment: enviroment, services: services)
+        try App.boot(app)
+        return app
+    }
+    
+    
+    func sendRequest<T: Content>(to path: String, method: HTTPMethod, headers: HTTPHeaders = HTTPHeaders(), body: T? = nil) throws -> Response {
+        let responder = try self.make(Responder.self)
+        let request = HTTPRequest(method: method, url: URL(string: path)!, headers: headers)
```

Comment:
Can **path** have a force unwrap?

Reply:
good catch. actually all path sent to this method should be valid (since this is just for test) but yeah this could improve.